src/content/blog/2024-02-21-eslint-2023-year-review.md

```diff
@@ -0,0 +1,99 @@
+
+## Updates to support the latest ECMAScript features
+
+The team spends much of the year anticipating new ECMAScript features and updating ESLint accordingly. 2023 was an outlier in that most of the changes weren't syntax-related, and so there wasn't a lot of additional work. The most significant syntax change, [hashbang comments](https://github.com/tc39/proposal-hashbang), is something that ESLint already handled to support Node.js.
+
+## Financials review
+
+The next few sections dig into the project financials. All of this information is gathered from the Open Collective transaction list with the exception of Tidelift payments (which are retrieved from Tidelift directly). Because transactions come in all throughout the course of a day, we decided to consider transactions beginning on January 1, 2023 at midnight UTC and ending on December 31, 2023 at 11:59 UTC as our fiscal 2023.
+
+### Income review
+
+During 2023, ESLint received $183,066.53 USD from all of our income sources. You can see the breakdown of our income sources in the following table.
+
+| **Source** | **Amount** |
+|-----------|-------------|
+| Open Collective | $120,801.30 |
+| GitHub Sponsors | $25,190.89 |
+| Tidelift | $30,077.84 |
+| Carbon Ads | $6,380.89 |
+| Thanks.dev | $524.61 |
+| Stackaid.us | $91.00 |
+| **Total** | **$183,066.53** |
+
+As usual, most of our donations came through Open Collective. Notably, our website ad revenue dropped roughly 50% from 2022, following an overall softening of the display ad business. We'll be looking at options to improve our website ad revenue in 2024.
+
+The newer sponsoring options, Thanks.dev and Stackaid.us, didn't provide much income but are both interesting ideas that we'll keep participating in.
+
+We are grateful to the companies and individuals who generously donated to support ESLint. We know that there has been a lot of turbulence in the tech industry in the past year, and we really appreciate everyone who still decided to sponsor ESLint.
+
+### Expenses review
+
+In 2023, we spent $185,702.36 USD on the ESLint project as a whole. This includes regular maintenance and development as well as other costs related to the project. See the table below for an overview.
+
+| **Category** | **Amount** |
+|-----------|-------------|
+| Maintenance and Development | -$66,910.37 |
+| TSC Stipend (Tidelift) | -$30,077.84 |
+| Triage and Support | -$10,307.78 |
+| Technical Writing | -$8,155.00 |
+| Contributor Pool | -$4,500.00 |
+| Health Insurance | -$13,447.41 |
+| Open Source Collective Fee | -$9,206.88 |
+| Ecosystem/Dependencies | -$24,400.00 |
+| Community Management | -$15,599.84 |
+| Payment Processor Fees | -$2,814.75 |
```

Comment:
I'm getting -$2,854.51. Not sure what might be causing the difference in calculations.

Reply:
I'm adding up everything in the column, but I think you're filtering out the rows that says "REFUNDED". Very confusing because the values in that column are also negative. I've updated my spreadsheet to match your values now.

Reply:
> but I think you're filtering out the rows that says "REFUNDED"

Yes, I noticed that OC covers payment processor fees for those transactions, so I think they don't affect the balance in any way and can be just filtered out.

https://opencollective.com/eslint/transactions?kind=PAYMENT_PROCESSOR_COVER